collectd-frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5.6-apache
```

Comment:
I'd say, use whichever web server works best with a given app. Nginx and Apache are not "yank one, plug another".